staging/src/k8s.io/apiserver/pkg/apis/flowcontrol/bootstrap/default.go

```diff
@@ -304,41 +303,14 @@ var (
 			},
 		},
 	)
-	SuggestedFlowSchemaSystemLeaderElection = newFlowSchema(
-		"system-leader-election", "leader-election", 100,
+	SuggestedFlowSchemaLeaderElection = newFlowSchema(
+		"leader-election", "leader-election", 100,
 		flowcontrol.FlowDistinguisherMethodByUserType,
 		flowcontrol.PolicyRulesWithSubjects{
 			Subjects: append(
 				users(user.KubeControllerManager, user.KubeScheduler),
 				kubeSystemServiceAccount(flowcontrol.NameAll)...),
 			ResourceRules: []flowcontrol.ResourcePolicyRule{
-				resourceRule(
-					[]string{"get", "create", "update"},
-					[]string{corev1.GroupName},
-					[]string{"endpoints", "configmaps"},
-					[]string{"kube-system"},
-					false),
```

Comment:
have kube-controller-manager / kube-scheduler dropped support for using endpoints or configmaps as leader election resources?

Reply:
for k-c-m and kube-scheduler, this is probably fine (though it needs an action-required release note for any users that haven't completed migrating those components to leases)

what was the difference between `system-leader-election` and `workload-leader-election` previously? I'm not seeing one...

Reply:
> what was the difference between system-leader-election and workload-leader-election previously? I'm not seeing one...

1. IIRC, we initially started with `system` being only for scheduler and kcm and workload being for all subjects.
But then apparently we added `kubeSystemServiceAccount(flowcontrol.NameAll),` to the `system` one too, which unified those.

2. But the actual difference that still existed before this PR is that in `system`, the endpoints/configmaps rule is only for kube-system namespace, whereas for `workload` it's for all namespaces.

Reply:
> the actual difference that still existed before this PR is that in system, the endpoints/configmaps rule is only for kube-system namespace, whereas for workload it's for all namespaces.

ah, now I see that

Reply:
> We migrated defaults to it couple releases ago.

then this part of the change seems fine.